src/crud-methods.js

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable */
+import setStorage from "./storage.js";
+
+export { deleteTask, updateStorage, tasks, addTask, editTask };
+
+const toDoList = document.getElementById("to-dos");
+
+class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+let tasks = [];
+const addTask = (taskValue) => {
+  const localData = JSON.parse(localStorage.getItem("todo"));
+  localData.map((i) => tasks.push(i));
+  const taskContainer = document.createElement("div");
+  taskContainer.className = "list-item";
+  taskContainer.innerHTML += `
+    <input type="checkbox" class="checkbox">
+    <span>${taskValue}</span>
+    <i class="fas fa-ellipsis-vertical"></i>
+    <i class="fas fa-trash-can"></i>
+  `;
```

Comment:
Hello @Cabraham1, you did very well, but it's just my suggestion to use the appendChild method rather than using innerHtml.